packages/sanity/src/structure/panes/documentList/DocumentSheetListPaginator.tsx

```diff
@@ -0,0 +1,71 @@
+/* eslint-disable i18next/no-literal-string */
```

Comment:
We don't have designs yet and we have high chances of removing pagination entirely, that why I avoid adding it to the translation resources